src/Core/CommandLine.cs

```diff
@@ -98,6 +98,7 @@ public static RootCommand BuildRootCommand()
             root.AddOption(EnvVarOption(new string[] { "--url-prefix" }, "URL prefix for ingress mapping", string.Empty));
             root.AddOption(EnvVarOption(new string[] { "--port" }, "Listen Port", 8080, 1, (64 * 1024) - 1));
             root.AddOption(EnvVarOption(new string[] { "--cache-duration", "-d" }, "Cache for duration (seconds)", 300, 1));
+            root.AddOption(EnvVarOption(new string[] { "--burst-service" }, "Service name for bursting metrics", "ngsa"));
```

Comment:
I feel like we should be able to reuse the app name from here:

```csharp

RootCommand root = new RootCommand            
{                
   Name = "Ngsa.Application", 
   Description = "NGSA Validation App",
   TreatUnmatchedTokensAsErrors = true,
};
```

Let's use that app name for the moment if it presents no issues and open another issue to discuss whether we should pass in the burst service name, derive it programmatically, or do something like pass in a suffix.

If it does present an issue, go with what you have, but open another issue as Design Review to look at this together.

Reply:
you should be able to extend Version to pull the info when we do the reflection for the version info

Reply:
agree that I wouldn't pass as param